Jenkinsfile

```diff
@@ -51,6 +52,15 @@ pipeline {
             }
         }
 
+        stage('Subsystem Test onprem') {
+            steps {
+                sh "make podman-pull-service-from-docker-daemon"
+                sh "export PODMAN_PULL_FLAG='--pull never'; make deploy-onprem-for-subsystem"
+                sh "export GINKGO_FOCUS_FLAG=-ginkgo.focus=minimal-set; make test-onprem"
+                sh "make clean-onprem || true"
```

Comment:
Can `clean-onprem` be included as part of `clear-all`? It would require fewer changes for the Jenkinsfile.
I'd like the cleaning to be running as post-action instead, which `clean-all` is already running.

Reply:
Yes clean-onprem can be included with clear-all. Done.